main/tests/server/src/com/google/refine/ClassResolverTests.java

```diff
@@ -0,0 +1,29 @@
+
+package com.google.refine;
+
+import org.testng.annotations.Test;
+
+import com.google.refine.preference.TopList;
+import com.google.refine.util.ParsingUtilities;
+import com.google.refine.util.TestUtils;
+
+public class ClassResolverTests {
+
+    String oldJson = "{\"class\":\"com.google.gridworks.preference.TopList\",\"top\":100,\"list\":[]}";
```

Comment:
Are there actually projects in the wild with this embedded class name? I didn't think that the "Google Refine" era and Jackson JSON era overlapped at all (and I thought there was no embedded class name prior to Jackson).

I'm going to see if I can dig up one or more vintage Google Refine / OpenRefine projects to test with.

Reply:
My bad! It looks like that class name has been buried in there since 2010. I still don't love it and would want to think through the various potential failure modes, particularly if it was used by a custom extension, as appears to have been the intention.